paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -83,7 +82,10 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
     }
 
     @Test
-    fun state_returnsCorrectSelectionForSavedPM() = runScenario(updateSelection = {}) {
+    fun state_returnsCorrectSelectionForSavedPM() = runScenario(
+        updateSelection = {},
+        formTypeForCode = { FormHelper.FormType.UserInteractionRequired },
```

Comment:
We're calling `formTypeForCode` in a lot more cases than before, so needed to update some existing tests.